### PR TITLE
Fix examples so they use the API guidelines

### PR DESCRIPTION
**Pull request by wltrup**
### TL;DR / Problem

The style guide, as cloned, has some examples that do not follow the [Swift API Guidelines](https://swift.org/documentation/api-design-guidelines/#follow-case-conventions).
### Solution

Make them follow the aforementioned guidelines. This PR also corrects some grammatical errors.

~~### Note ###~~

~~This PR only does what it says above up to the section **Classes and Structures**. Please do not merge this PR just yet as there is more to come.~~

### CODE REVIEW

File: style-guides/swift/README.markdown [hunk 1]
```diff
@@ -97,32 +97,32 @@ let userId: UserId
 For functions and init methods, prefer named parameters for all arguments unless the context is very clear. Include external parameter names if it makes function calls more readable.
 
 ```swift
-func dateFromString(dateString: String) -> NSDate
-func convertPointAt(column column: Int, row: Int) -> CGPoint
-func timedAction(afterDelay delay: NSTimeInterval, perform action: SKAction) -> SKAction!
+func date(from source: String) -> NSDate
```
Comment (dominicfreeston) [inline style-guides/swift/README.markdown]: why `from source` and not `from string`?

File: style-guides/swift/README.markdown [hunk 2]
```diff
@@ -97,32 +97,32 @@ let userId: UserId
...
+func pointAt(column column: Int, row: Int) -> CGPoint
```
Comment (dominicfreeston) [inline style-guides/swift/README.markdown]: would `convertedPointAt` be a more direct translation of `convertPointAt`? Hard to tell without context tbh. And I guess not super critical.

File: style-guides/swift/README.markdown [hunk 3]
```diff
@@ -387,7 +386,7 @@ class Circle: Shape {
 The example above demonstrates the following style guidelines:
 
  + Specify types for properties, variables, constants, argument declarations and other statements with a space after the colon but not before, e.g. `x: Int`, and `Circle: Shape`.
- + Define multiple variables and structures on a single line if they share a common purpose / context.
+ + Define multiple variables and structures on separate lines even if they share a common purpose / context.
```
Comment (dominicfreeston) [inline style-guides/swift/README.markdown]: 👍 but that possibly deserves its own PR? Or maybe highlighting is here enough!

File: style-guides/swift/README.markdown (hunk 1 above)
Comment (wltrup) [inline style-guides/swift/README.markdown]: Because the fact that `source` is a string is already clear from the argument type. We could potentially have another function `date(from source:)` where the source would be of another type.

Comment (wltrup) [inline style-guides/swift/README.markdown]: Oh, wait, you're talking about the internal name. Yeah, I suppose we could still keep it `string`. I prefer `source` though. :)

File: style-guides/swift/README.markdown (hunk 2 above)
Comment (wltrup) [inline style-guides/swift/README.markdown]: Yeah, hard to tell but it seems to me that the word `converted` is not necessary. As you said, though, not a major issue.

File: style-guides/swift/README.markdown (hunk 3 above)
Comment (wltrup) [inline style-guides/swift/README.markdown]: Once again, I originally meant this to be on a separate PR. TBH, at this point I'd prefer not to create yet another PR just for this so if people are ok with this, I'd like to keep it as is. Will defer to what people want, though.

**Comment by wltrup** [comment]
I'm taking away the 'do not merge' sign as there is enough in this PR already. Will create new PRs if/when needed for badly styled examples.
